Tests/SonarScanner.MSBuild.PostProcessor.Test/PostProcessorTests.cs

```diff
@@ -332,10 +330,10 @@ public void Execute_TeamBuild_CoverageReportProcessorCalled()
         AssertProcessCoverageReportsCalledIfNetFramework();
 
 #if NETFRAMEWORK
-        fileWrapper.Received().AppendAllText(
+        runtime.File.Received().AppendAllText(
```

Comment:
I'm not a big fan of this. You need to know that `runtime` is a `TestRuntime` and that `TestRuntime` NSubstitutes `IFileWrapper` so you can call NSubstitutes `Received` here. That is a bit too much indirection for my taste. This is also true for the setup calls later.